notebook/services/contents/filemanager.py

```diff
@@ -94,43 +93,6 @@ def _update_save_script(self):
 
         self.post_save_hook = _post_save_script
 
-    post_save_hook = Any(None, config=True, allow_none=True,
-        help="""Python callable or importstring thereof
-
-        to be called on the path of a file just saved.
-
-        This can be used to process the file on disk,
-        such as converting the notebook to a script or HTML via nbconvert.
-
-        It will be called as (all arguments passed by keyword)::
-
-            hook(os_path=os_path, model=model, contents_manager=instance)
-
-        - path: the filesystem path to the file just written
-        - model: the model representing the file
-        - contents_manager: this ContentsManager instance
-        """
-    )
-
-    @validate('post_save_hook')
-    def _validate_post_save_hook(self, proposal):
-        value = proposal['value']
-        if isinstance(value, string_types):
-            value = import_item(value)
-        if not callable(value):
-            raise TraitError("post_save_hook must be callable")
-        return value
-
-    def run_post_save_hook(self, model, os_path):
-        """Run the post-save hook if defined, and log errors"""
-        if self.post_save_hook:
-            try:
-                self.log.debug("Running post-save hook on %s", os_path)
-                self.post_save_hook(os_path=os_path, model=model, contents_manager=self)
```

Comment:
post_save hook was declared here because the signature requires that the result be a file on disk. The higher-level manager doesn't have a notion of `os_path`, so the same post_save hook doesn't make sense.